hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java

```diff
@@ -72,9 +74,10 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
 
         final var transactionBody = context.body().fileAppendOrThrow();
         final var fileStore = context.createStore(ReadableFileStore.class);
-        final var fileMeta = preValidate(transactionBody.fileID(), fileStore, context, false);
+        preValidate(transactionBody.fileID(), fileStore, context, false);
```

Comment:
Shouldn't these checks be done in handle ? In all handlers. It is possible configuration or state changed from prehandle to handle

Reply:
that's why it checked in both